doc/conf.py

```diff
@@ -25,7 +25,7 @@
 author = u'Zeek Project'
 
 version = subprocess.check_output("../scripts/autogen-version").decode("utf8")
-release = version
+release = "1.0.0" # most recent release version
```

Comment:
Updating this should end up in a release checklist.

Reply:
https://github.com/zeek/spicy/wiki/Release-Checklist